test/powershell/Language/Operators/NullConditional.Tests.ps1

```diff
@@ -260,6 +260,15 @@ Describe 'NullConditionalMemberAccess' -Tag 'CI' {
             $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
             $PSDefaultParameterValues["it:skip"] = $true
         }
+
+        function ExecuteTestIfFeatureIsEnabled([string] $TestContet)
```

Comment:
You need to declare the experimental feature tests in `test/testmetadata.json` file, so that they will run in our release test automation.

Reply:
Fixed